frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart

```diff
@@ -80,6 +81,15 @@ class _AppFlowyEditorPageState extends State<AppFlowyEditorPage> {
   late final styleCustomizer = EditorStyleCustomizer(context: context);
   DocumentBloc get documentBloc => context.read<DocumentBloc>();
 
+  @override
+  void initState() {
+    super.initState();
+    commandShortcutEvents.insert(
+      0,
+      showEmojiPickerEvent(context),
+    );
```

Comment:
You don't really care about the index, just use `.add`:
```suggestion
    commandShortcutEvents.add(showEmojiPickerEvent(context));
```